Makefile

```diff
@@ -114,21 +118,28 @@ e2e.test:
 test-e2e: e2e.test
 	./e2e.test -test.timeout 180m -test.v -ginkgo.v
 
-test-go: generate
-	go build -tags containers_image_openpgp ./...
+test-go: generate build-all validate-go lint-go unit-test-go coverage-go
 
+.PHONY: validate-go
+validate-go:
 	gofmt -s -w cmd hack pkg test
 	go run ./vendor/golang.org/x/tools/cmd/goimports -w -local=github.com/Azure/ARO-RP cmd hack pkg test
 	go run ./hack/validate-imports cmd hack pkg test
 	go run ./hack/licenses
 	@[ -z "$$(ls pkg/util/*.go 2>/dev/null)" ] || (echo error: go files are not allowed in pkg/util, use a subpackage; exit 1)
 	@[ -z "$$(find -name "*:*")" ] || (echo error: filenames with colons are not allowed on Windows, please rename; exit 1)
 	@sha256sum --quiet -c .sha256sum || (echo error: client library is stale, please run make client; exit 1)
-	go test -tags e2e -run ^$$ ./test/e2e/...
-
 	go vet ./...
-	set -o pipefail && go test -v ./... -coverprofile cover.out | tee uts.txt
 
+.PHONY: unit-test-go
+unit-test-go:
+	go run ./vendor/gotest.tools/gotestsum/main.go --format pkgname --junitfile report.xml
```

Comment:
Do I understand correctly that with `--format pkgname` we will see just package names in case of skipped/green tests. But if something fails, we will see package name and name of the failed test(s)?

Reply:
Yes, the package output applies for passed tests. When a test fails, it is outputted with full verbosity.